automation/BuildRegistrationDefinition/registration_definition.template

```diff
@@ -0,0 +1,21 @@
+{
+   "repository_name": "",
+   "docker_username": "",
+   "docker_password": "",
+   "envs_whitelist": [
+      "RUNQ_ROOTDISK",
+      "RUNQ_RUNQENV",
+      "RUNQ_SYSTEMD",
+      "IMAGE_TAG",
+      "REGION",
+      "PHASE",
+      "LPAR_NAME",
+      "CPC",
+      "RUNQ_CPU",
+      "RUNQ_MEM",
+      "POD"
+   ],
+   "public_key_id": "",
+   "public_key": "",
+   "vendor_key": ""
+}
```

Comment:
We should ask Ryley or someone to help add a comment to document what each of those is or the spec that defined these variables if there is such a public document already :)

Reply:
there is information in README.md 

`The Keys

To sign and deploy an image to HPVS, we need three to four sets of keys:

Root key; root of content trust for an image tag
Repository key (aka "target key"); allows you to sign image tags and manage delegations (including delegated keys or permitted delegation paths)
Encryption key for the HPVS registration definition file
Vendor key, required for signing the HPVS registration definition file
Delegation key (optional); allows you to delegate signing image tags to other publishers without having to share your repository key`